app/assets/javascripts/discourse/app/controllers/full-page-search.js

```diff
@@ -464,9 +479,15 @@ export default Controller.extend({
     });
   },
 
+  @action
+  recheckSearchResults() {
+    // Note: Action is called by pluginAPI method `addSearchResults()`
+    this.notifyPropertyChange("searchResultPosts");
+  },
+
```

Comment:
To avoid the need for this, we could:

1. Make `additionalSearchResults` a TrackedArray

2. Convert `searchResultPosts` to a native getter, so that it automatically re-evaluates when any trackable state changes